sql/deploy/add-device-triton-setup.sql

```diff
@@ -1,5 +1,5 @@
-SELECT run_migration(18, $$
+BEGIN;
```

Comment:
Since I doubt these old migrations will even run anymore (and some do naughty things with data presuming the presence of old data that won't exist on a fresh db), do we need to keep most of these files at all? We could pick a line in the sand like the 2.24 release and forget about all migrations before that point, and start with that release's sql/schema.sql instead.  Migrations after that point will need to be written with sqitch in mind in case there are any other things to consider (are there?)

Reply:
I'm fine with that idea but I didn't want to bite *that* discussion off in what is mostly a simple transition of underlying tooling. I'd rather do that in a separate PR.

Reply:
So what I've done (and I'm happy to re-visit this if you think it's warranted) is I've put as my last commit a migration that purges the old migrations table (if it exists). The rollback for this migration throws an exception and stops everything before it from happening.

I've done some minimal work to make sure you _could_ rollback before this commit but I haven't gone out of my way to make sure that it leaves the application in a sane state. Simply that it attempts to roll things back such that you can process reversions before that relatively cleanly. (For example I don't attempt to add-back dropped columns, or drop new columns, I don't attempt to add back all the purged Workflow tables etc.)

This means that it the history of these changes is minimally preserved in Sqitch, and if necessary we _can_ re-visit these migrations (rework them in sqitch terms etc). But by default they're gated behind a line in the sand.